tests/src/kakarot/accounts/test_account_contract.py

```diff
@@ -367,6 +367,30 @@ def test_should_raise_with_wrong_signature(self, cairo_run):
                     chain_id=CHAIN_ID,
                 )
 
+        @composite
+        def draw_signature_not_in_range(draw):
+            # create signature with 4 elements < 2 ** 128 and one > 2 ** 128
+            signature = [
+                draw(integers(min_value=0, max_value=2**128 - 1)) for _ in range(4)
```

Comment:
not using `lists` ?

Reply:
good catch thank you, done with [54fb57e](https://github.com/kkrt-labs/kakarot/pull/1331/commits/54fb57e34b38fd5c2160e97766e3fa076219eb9f)